client/src/pages/UpdateTaskPage.tsx

```diff
@@ -50,23 +45,13 @@ export const UpdateTaskPage: React.FC<RouteComponentProps<IUpdateMatchParams>> =
     setShowToast(true);
   }
 
-  const submit = (event: SyntheticEvent) => {
-    event.preventDefault();
-    const task = data.findTasks;
-    delete task.__typename;
-    const variables = { 
-      input: {
-        ...task,
-        title: title || task.title,
-        description: description || task.description,
-      }
-    };
-
+  const submit = (model: any) => {
+    delete model.__typename;
```

Comment:
`onSubmit` prop (`handleSubmit` of `TaskForm`) shouldn't mutate the passed model as it may be (here it actually is) a state reference.

Reply:
Thanks, that is good to know. @radekmie, thanks for your feedback on this PR, I really appreciate it. uniforms is awesome btw